sphinx/texinputs/sphinxlatexnumfig.sty

```diff
@@ -61,7 +61,7 @@
     \ifspx@opt@mathnumfig
       \@addtoreset{equation}{section}%
     \fi%
-    \g@addto@macro\spx@preAthefigure{\ifnum\c@section>\z@\arabic{section}.}%
+    \g@addto@macro\spx@preAthefigure{\ifnum\c@section>\z@.\arabic{section}}%
```

Comment:
```suggestion
   \g@addto@macro\spx@preAthefigure{\ifnum\c@section>\z@\ltx@ifundefined{c@chapter}{}{.}\arabic{section}}
```

`'howto'` class does not use chapters only sections.  

But we may have a problem perhaps with `latex_toplevel_sectioning='part'`.  I need to check later.

Reply:
I am using this patch with a large (2000-page) document with parts.

Reply:
Was your document using `latex_theme='howto'`?